WeScanTests/VisionRectangleDetectorTests.swift

```diff
@@ -16,6 +16,36 @@ final class VisionRectangleDetectorTests: FBSnapshotTestCase {
     super.setUp()
     recordMode = false
   }
+
+  /// Function gathered from [here](https://stackoverflow.com/questions/44462087/how-to-convert-a-uiimage-to-a-cvpixelbuffer) to convert UIImage to CVPixelBuffer
+  ///
+  /// - Parameter image: image to be converted
+  /// - Returns: new CVPixelBuffer
+  private func buffer(from image: UIImage) -> CVPixelBuffer? {
```

Comment:
Can you add this to an extension in this class, so the main class body only contains tests? In that way we keep readability combined with adding new functionalities

Reply:
Totally makes sense, I'll add that tomorrow when I'm at my laptop, along with the other small bits.